app/styles/new-ui.scss

```
@@ -0,0 +1,84 @@
+.new-ui {
+  // to accommodate and offset old ui margins
+  margin-right: -15px;
+  margin-left: -15px;
+  height: 100%;
+  // height: 100vh;
+
+  .grid-container {
+    display: grid;
+    height: 100%;
+
+    grid-column-gap: 0px;
+    grid-row-gap: 0px;
+
+    grid-template-columns: 50px;
+    grid-template-rows: auto;
+
+    grid-template-areas: 'left right';
+
+    .grid-item.left {
+      background-color: #fff;
+      border-left: 1px solid #f2f2f2;
+      border-right: 1px solid #f2f2f2;
+
+      grid-area: left;
+    }
+
+    .grid-item.right {
+      grid-area: right;
+
+      display: grid;
+      grid-column-gap: 0px;
+      grid-row-gap: 0px;
+
+      grid-template-areas:
+        'pipeline-header  pipeline-header  pipeline-header        pipeline-header        pipeline-header'
+        'pipeline-tab     pipeline-tab     pipeline-tab           pipeline-tab           pipeline-tab'
+        'pipeline-content pipeline-content pipeline-workflowgraph pipeline-workflowgraph pipeline-workflowgraph';
```

```suggestion
      grid-template-areas:
        'pipeline-header  pipeline-header'
        'pipeline-tab     pipeline-tab'
        'pipeline-content pipeline-workflowgraph';
```
This should make it easier to read and then you can set the width of `pipeline-content` and `pipeline-workflowgraph`.